astroquery/mast/core.py

```diff
@@ -500,6 +687,150 @@ def service_request_async(self, service, params, pagesize=None, page=None, **kwa
 
         return response
 
+    @class_or_instance
+    def catalogs_service_request_async(self, service, params, page_size=None, page=None, **kwargs):
+        """
+           Given a MAST fabric service and parameters, builds and excecutes a fabric microservice catalog query.
```

Comment:
the docstring should be indented the same amount as the code (note that the closing """s are indented below)

Reply:
Updated indentation